diffcp/cone_program.py

```diff
@@ -48,6 +48,10 @@ def solve_and_derivative_batch(As, bs, cs, cone_dicts, n_jobs=-1, warm_starts=No
     return pool.starmap(solve_and_derivative_wrapper, args)
 
 
+class SolverError(Exception):
+    pass
+
+
 def solve_and_derivative(A, b, c, cone_dict, warm_start=None, **kwargs):
```

Comment:
Document the behavior for infeasible/unbounded problems in the README, please? :)